components/Atoms/Button/Button.md

```diff
@@ -1,8 +1,14 @@
-This button is really basic.
-
+Un bouton avec les propriétés par defaut:
 ```tsx
-<Button styleClass="btn">Hello mate</Button>
+<Button url="#" styleClass="btn">Bouton</Button>
 ```
+les autres styles de boutons:
 ```tsx
-<Button styleClass="btn--danger">Hello danger</Button>
+<Button url="#" styleClass="btn--danger">Danger</Button>
```

Comment:
Ici tu aurais pu utiliser `(j,t)sx padded` pour les avoir côte à côte espacés

Reply:
https://github.com/styleguidist/react-styleguidist/blob/master/docs/Documenting.md#usage-examples-and-readme-files